image/registry_handler.go

```diff
@@ -58,13 +56,11 @@ TODO: This is a temporary solution in order to get insecure registries in other
 TODO: Ideally we should fix the `imgutil.options` struct visibility in order to mock and test the `remote.WithRegistrySetting`
 TODO: function correctly and use the RegistryHandler everywhere it is needed.
 */
-func GetInsecureOptions(insecureRegistries []string, imageRef string) []remote.ImageOption {
+func GetInsecureOptions(insecureRegistries []string) []remote.ImageOption {
 	var opts []remote.ImageOption
 	if len(insecureRegistries) > 0 {
```

Comment:
I think we can remove this redundant if len check. If there is nothing in there it won't loop anyway

Reply:
Ops you are right, fixing it 😄